### PR TITLE
List the beers

### PR DESCRIPTION
**Pull request by samson84**
Show the users the list of all beer names on the console.

### CODE REVIEW

**Comment by Sarfeher** [comment]
dsjldsajlda

File: showAll.js [hunk 1]
```diff
@@ -6,6 +6,11 @@
 const getBeers = require('./data')
 
 function main() {
+  const beeers = getBeers()
```
Comment (Sarfeher) [inline showAll.js]: tipo in const name

Comment (Sarfeher) [inline showAll.js]: Awsome

File: showAll.js [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
+
+  for (const beer of beeers) {
+    console.log(`Beer: ${beer.name}`)
```
Comment (attila-kerekes) [inline showAll.js]: I like the additional "Beer:" explanation.

**Review by Sarfeher — APPROVED** [review]

**Review by attila-kerekes — CHANGES_REQUESTED** [review]

**Review by attila-kerekes — APPROVED** [review]

**Pull request merged**